test/utilities/oas-validator.test.ts

```diff
@@ -112,6 +112,156 @@ describe('OasValidator', () => {
       });
     });
 
+    describe('both content and schema exist on parameter', () => {
+      it('returns a validation failure', async () => {
+        const schema = generateMockSchema([
+          {
+            name: 'gryffindor',
+            example: 'ginny weasly',
+            schema: {
+              type: 'string',
+              description: 'founded by Godric Gryffindor',
+            },
+            content: {
+              'document/howler': {
+                schema: {
+                  type: 'string',
+                  example: 'ron weasly',
+                },
+              },
+            },
+          },
+        ]);
+
+        const validator = new OasValidator((schema as unknown) as OasSchema);
+
+        const failures = await validator.validateParameters('operation1', {
+          gryffindor: 'luna lovegood',
+        });
+
+        expect(failures).toHaveLength(1);
+        expect(failures).toContainValidationFailure(
+          'Parameter object must have either schema or content set, but not both. Path: parameters -> gryffindor',
+        );
+      });
+    });
+
+    describe('content contains more than one entry', () => {
+      it('returns a validation failure', async () => {
+        const schema = generateMockSchema([
+          {
+            name: 'magical deliveries',
+            example: 'the daily prophet',
+            content: {
+              'document/howler': {
+                schema: {
+                  type: 'string',
+                  example: 'petunia dursley',
+                },
+              },
+              'owl/package': {
+                schema: {
+                  type: 'string',
+                  example: 'nimbus 2000',
+                },
+              },
+            },
+          },
+        ]);
+
+        const validator = new OasValidator((schema as unknown) as OasSchema);
+
+        const failures = await validator.validateParameters('operation1', {
+          'magical deliveries': 'firebolt',
+        });
+
+        expect(failures).toHaveLength(1);
+        expect(failures).toContainValidationFailure(
+          'Parameter content object should only have one key. Path: parameters -> magical deliveries -> content',
+        );
+      });
+    });
+
+    describe('content does not contain a schema object', () => {
+      it('returns a validation failure', async () => {
+        const schema = generateMockSchema([
+          {
+            name: 'magical deliveries',
+            example: 'the daily prophet',
+            content: {
+              'document/howler': {
+                properties: {
```

Comment:
Can we change this to something more explicitly fake? I think properties looks like it could be a correct key. (I think this supposed to be a [Media Type Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#mediaTypeObject), but I could be completely off)